packages/jyve-pyodide-unsafe-extension/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "author": "Dead Pixels Collective",
+  "bugs": {
+    "url": "https://github.com/deathbeds/jyve/issues"
+  },
+  "dependencies": {
+    "@deathbeds/jyve-pyodide-unsafe": "^0.4.0",
+    "@jupyterlab/application": "^0.16.2",
+    "tslib": "^1.9.0"
+  },
+  "description": "An extension for a totally unsafe browser pyodide kernel",
+  "files": [
+    "{lib,style}/**/*.{css,d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}"
+  ],
+  "homepage": "https://github.com/deathbeds/jyve",
+  "jupyterlab": {
+    "extension": true
+  },
+  "jyve": {
+    "help_links": [
+      {
+        "text": "Pyodide",
+        "url": "https://github.com/iodide-project/pyodide"
+      }
+    ],
+    "kernelspec": {
+      "arvg": [
+        "echo",
+        "jyve"
+      ],
+      "display_name": "Pyodide (unsafe) — Jyve",
+      "language": "python",
+      "name": "jyve-pyodide-unsafe",
+      "resources": {
+      }
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "python",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "python3",
+      "version": "ES2015"
```

Comment:
hangover (not your fault)... really want to have these reflect the version of the implementation a la #10... and not duplicate them

Reply:
So this version is supposed to be the version of Pyodide, or of the pyodide-unsafe-extension here...?  Haven't made pyodide releases yet, but this will likely be a combination of the underlying CPython version and the pyodide-specific bits version.